x-pack/test/rule_registry/security_and_spaces/tests/basic/get_browser_fields_by_feature_id.ts

```diff
@@ -46,7 +46,16 @@ export default ({ getService }: FtrProviderContext) => {
           'logs',
           'uptime',
         ]);
-        expect(Object.keys(browserFields)).to.eql(['base', 'event', 'kibana', 'message']);
+        expect(Object.keys(browserFields)).to.eql([
```

Comment:
Additional browser fields because the alerts indices are installed on Kibana start so they contain mappings outside of those loaded by the esArchiver data.